src/Uno.UWP/System/Profile/AnalyticsInfo.Android.cs

```diff
@@ -43,14 +43,14 @@ private static UnoDeviceForm GetDeviceForm()
 			return UnoDeviceForm.Desktop;
 		}
 
-		if (SafeFormCheck(IsPhone))
+		if (SafeFormCheck(IsTablet))
 		{
-			return UnoDeviceForm.Mobile;
+			return UnoDeviceForm.Tablet;
 		}
 
-		if (SafeFormCheck(IsTablet))
+		if (SafeFormCheck(IsPhone))
 		{
-			return UnoDeviceForm.Tablet;
+			return UnoDeviceForm.Mobile;
```

Comment:
Determining that a device is a phone or a table has always been difficult, and in this case, changing this will be a breaking change because there are phones are that over 6 inches in diagnonal... (https://www.techradar.com/reviews/xiaomi-13#section-xiaomi-13-review-specs). Let's keep the original behavior for now as it will break for large phones, unless we can get some sort of OEM provided hint that a device really is a tablet or not.

Reply:
I don't think there is any reliable way to know whether the device is phone or tablet, unfortunately.

@jeromelaban Can't we adjust the numbers as necessary to get a reasonable behavior? currently we will always be returning mobile for both phones and tablets.

Reply:
Sure. Let's mark this change as breaking though.